File: source/baselib/src/main/java/com/hopechart/baselib/utils/Logs.java
package com.hopechart.baselib.utils;

import android.text.TextUtils;
import android.util.Log;

import com.hopechart.baselib.BaseApplication;

public class Logs {

    public static String currentTag = "";

    public static void e(String message) {
        String packageName = BaseApplication.getInstance().getPackageName();
        e(packageName, message);
    }

    public static void e(String tag, String message) {
        Log.e(tag, message);
    }

    public static void tagE(String message) {
        if (!TextUtils.isEmpty(currentTag))
            e(currentTag, message);
        else
            e(message);
    }
}

File: source/baselib/src/main/java/com/hopechart/baselib/utils/Logs.kt
package com.hopechart.baselib.utils

import android.util.Log
import com.hopechart.baselib.BaseApplication

class Logs {
    companion object {
        @JvmStatic
        fun e(tag: String, content: String) {
            Log.e(tag, content)
        }

        @JvmStatic
        fun e(content: String) {
            e(BaseApplication.getInstance().packageName, content)
        }
    }
}